webapp/src/components/AuctionPage/TokenDropdown/TokenDropdown.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { Dropdown } from 'semantic-ui-react'
+
+import { SYMBOLS } from 'modules/auction/utils'
+
+import './TokenDropdown.css'
+
+const OPTIONS = SYMBOLS.map(symbol => ({
+  text: symbol,
+  value: symbol
+}))
+
+export default class TokenDropdown extends React.PureComponent {
```

Comment:
_Not critical_, can we add PropTypes here too?

Reply:
✅ done